common/plugins/eu.esdihumboldt.hale.common.test/Tests.product

```diff
@@ -251,6 +251,7 @@
       <plugin id="eu.esdihumboldt.util.orient.embedded.test"/>
       <plugin id="eu.esdihumboldt.util.resource"/>
       <plugin id="eu.esdihumboldt.util.resource.schemas.citygml"/>
+      <plugin id="eu.esdihumboldt.util.resource.schemas.end"/>
```

Comment:
Does not really need to be added here, as it is not used in any tests, but also does not hurt.

Reply:
I have left it unchanged.